### DISCUSSION TITLE
多wan如何设置优先级策略

### DISCUSSION THREAD

**Discussion by chqyhslwszds** [istoreos/istoreos#1977] [Q&A]
r3s设备增加了4g网卡
lan口平时作为旁路模式，接收4g口zerotier流量进行远程局域网管理
但一旦插上有线wan口后，系统流量默认就切换到wan口，且不会因为wan口没有流量自动切回。
需要的策略是：
4g口优先wan口
两者都能上网时，只要4g有信号或者能上网，一切流量优先走4g。
4g没有信号时自动走wan口，但一旦4g流量恢复，又自动走4g。
谢谢！

**Comment by jjm2473** [comment]
如果4G口没有信号是会自动下线接口的话，一般设置接口的metric就行了，越大的值优先级越低。
如果4G口不管有没有信号，接口都是在线的，需要额外判断4G是不是有效，那就只能自己写脚本自动下线4G口了。

**Reply by chqyhslwszds** [reply]
收到，测试跃点成功，谢谢
